features/post.feature

```diff
@@ -8,6 +8,17 @@ Feature: Manage WordPress posts
     Then STDOUT should be a number
     And save STDOUT as {POST_ID}
 
+    When I run `wp post exists {POST_ID}`
+    Then STDOUT should be:
+      """
+    Success: Post with ID {POST_ID} exists.
+      """
+    And the return code should be 0
+
+    When I run `wp post exists 1000`
```

Comment:
As you expect this call to return an error (non-null exit code), you need to use `When I try...`, not `When I run...`